CHANGELOG.rst

```diff
@@ -5,7 +5,94 @@ Changelog
 TBR
 ---
 
-* Official support for Python 3.11
+This release enables scrapy-poet to fully support item types as dependencies in
+page objects and spider callbacks. The following is now possible:
+ 
+.. code-block:: python
+
+    import attrs
+    import scrapy
+    from web_poet import WebPage, handle_urls, field
+
+    @attrs.define
+    class Image:
+        url: str
+
+    @handle_urls("example.com")
+    class ProductImagePage(WebPage[Image]):
+        @field
+        def url(self) -> str:
+            return self.css("#product img ::attr(href)").get("")
+
+    @attrs.define
+    class Product:
+        name: str
+        image: Image
+
+    @handle_urls("example.com")
+    @attrs.define
+    class ProductPage(WebPage[Product]):
+        # ✨ NEW: Notice that the PO can ask for items as dependencies.
+        # An instance of ``Image`` is injected behind the scenes by calling the
+        # ``.to_item()`` method of ``ProductImagePage``.
+        image: Image
+
+        @field
+        def name(self) -> str:
+            return self.css("h1.name ::text").get("")
+
+        @field
+        def image(self) -> Image:
+            return self.image
```

Comment:
is this example working? Isn't self.image redefined to be a method?

Reply:
Even if not, probbaly it makes sense to change it, to avoid confusion

Reply:
Ohh yeah, good catch!